django/db/backends/sqlite3/schema.py

```diff
@@ -70,7 +70,7 @@ def _remake_table(self, model, create_fields=[], delete_fields=[], alter_fields=
         for field in create_fields:
             body[field.name] = field
             # Choose a default and insert it into the copy map
-            if not isinstance(field, ManyToManyField):
+            if field.get_internal_type() != 'ManyToManyField':
```

Comment:
I commented to that effect on the ticket. Only problem is we don't have that flag on 1.7.